kalliope/__init__.py

```diff
@@ -195,3 +208,38 @@ def configure_logging(debug=None):
         logger.setLevel(logging.INFO)
 
     logger.debug("Logger ready")
+
+
+def get_list_signal_class_to_load(brain):
```

Comment:
This Algo could be replaced by a [python set](https://docs.python.org/2/library/sets.html)

Reply:
Done